package.json

```
@@ -45,28 +46,28 @@
   ],
   "license": "MIT",
   "devDependencies": {
-    "@rollup/plugin-commonjs": "^23.0.2",
-    "@rollup/plugin-json": "^5.0.1",
-    "@rollup/plugin-node-resolve": "^15.0.1",
+    "@rollup/plugin-commonjs": "^28.0.6",
+    "@rollup/plugin-json": "^6.1.0",
+    "@rollup/plugin-node-resolve": "^16.0.1",
+    "@rollup/plugin-terser": "^0.4.4",
     "chai": "^4.3.7",
     "cmof-parser": "^0.5.1",
-    "eslint": "^9.12.0",
+    "eslint": "^9.19.0",
     "eslint-plugin-bpmn-io": "^2.0.2",
-    "esm": "^3.2.25",
     "execa": "^5.0.0",
     "glob": "^7.1.6",
     "jsondiffpatch": "^0.4.1",
     "mocha": "^8.4.0",
     "npm-run-all": "^4.1.5",
-    "rollup": "^2.79.1",
-    "rollup-plugin-terser": "^7.0.2",
+    "rollup": "^4.44.1",
     "sax": "^1.2.4",
     "tiny-stack": "^2.0.1",
     "xsd-schema-validator": "^0.7.0"
   },
   "dependencies": {
     "min-dash": "^4.0.0",
-    "moddle": "^6.0.0",
-    "moddle-xml": "^10.0.0"
+    "moddle": "^7.0.0",
+    "moddle-xml": "^11.0.0",
+    "rollup-plugin-generate-package-json": "^3.2.0"
```

```suggestion
    "rollup-plugin-generate-package-json": "^3.2.0"
```

Why? What does this accomplish?

---

Easy solution to generate package.json with `"type": "commonjs"` to fix UMD bundle test and ensure it works as expected

---

> Easy solution to generate package.json with "type": "commonjs"

Let's migrate to ESM, drop UMD. Today every environment supports ES modules, UMD is no longer needed, and where it is a user can generate a UMD-wrapper themselves.

---

dropped UMD